howto/change-python.md

```diff
@@ -0,0 +1,87 @@
+# Bump Python Version for DjangoGirls.org Website
+
+In order to bump up the Python version for the DjangoGirls.org website, you 
+need to have access to our PythonAnywhere account, and follow these instructions:
+
+[Login to PythonAnywhere](https://www.pythonanywhere.com/user/djangogirls2/consoles/). 
+* Credentials can be found in 1Password.
+
+* Check if the Python version you want to upgrade to is supported by the current
+system image used on PythonAnywhere. 
+You can do so by clicking on the pen icon on the Web Tab or by 
+[reading this](https://help.pythonanywhere.com/pages/ChangingSystemImage#available-python-versions-for-system-images).
+ If the version of Python is already supported by the current image, there is 
+ no need to change the system image. 
+ If the version is not supported by the current system image, then the system
+ image needs to be upgraded. 
+
+* Check which `node` version we are using opening the `Bash Console` and 
+type this command
+
+  ```which node```
+
+  Write down the version of `node` you get from this step and save it for later 
+use. You will need to reinstall it in the new image. (Skip this if image stays the same)
+
+* Change the system image following 
+[these instructions](https://help.pythonanywhere.com/pages/ChangingSystemImage#available-python-versions-for-system-images). 
+(skip this step if version is supported by current image).
+
+* Navigate to the `.virtualenvs` folder by typing 
+
+    ```cd .virtualenvs```
+    
+* Create a new virtual environment. Make sure to specify the Python version in 
+the command. Replace `*` with specific version number.
+
+    ```python3.* -m venv new-venv-name```
```

Comment:
If we want syntax highlighting:
````suggestion
    ```bash
    python3.* -m venv new-venv-name
    ```
````